workflowhelpers/internal/cf_auth_test.go

```diff
@@ -68,13 +69,30 @@ var _ = Describe("cf auth", func() {
 			})
 
 			It("errors", func() {
-				err := CfAuth(cmdStarter, redactingReporter, "user", password, 5*time.Second)
+				err := CfAuth(cmdStarter, redactingReporter, "user", password, origin, 5*time.Second)
 
 				Expect(err).To(HaveOccurred())
 				Expect(err).To(MatchError("something went wrong"))
 			})
 		})
 
+		Context("when the origin is set", func() {
+			BeforeEach(func() {
+				origin = "origin"
+			})
+
+			AfterEach(func() {
+				origin = ""
```

Comment:
I don't _think_ this `AfterEach()` is necessary.

Reply:
Not if it is initialized at the correct place. Done!